app/Entry.js

```diff
@@ -66,7 +67,9 @@ const ExportStack = () => (
       ...SCREEN_OPTIONS,
       cardStyleInterpolator: fade,
       gestureEnabled: false,
-    }}>
+    }}
+    initialRouteName={isGPS ? 'ExportSelectHA' : 'ExportIntro'}>
```

Comment:
For sake of the demo dont actually do this, but, do you think its possible to consolidate the flows between gps and bte such that they have the same number of screens?

Reply:
I think this is a question for design, but I remember someone saying this is just a temporary flow for the demo. I didn't want to do too much here bc of that.